releases/release-1.27/release-notes/maps/pr-114418-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 114418
+releasenote:
+  text: New feature gate, `ServiceNodePortStaticSubrange`, to enable the new strategy
```

Comment:
`Added new feature gate ....` might read better. Please check and correct if possible.